Dockerfile

```diff
@@ -1 +1,76 @@
+# Use ARG to define the base image tag at build time (default to Ubuntu 20.04)
+ARG UBUNTU_VERSION=latest
+FROM ubuntu:${UBUNTU_VERSION}
 
+# Set the company as the image owner
+LABEL maintainer="DEVOPS EASY LEARNING <devops@example.com>"
+
+# Install the required packages
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    ansible \
+    curl \
+    git \
+    gnupg \
+    jq \
+    linux-headers-$(uname -r) \
+    openssh-client \
+    postgresql-client \
+    python3 \
+    kubectl \
+    kubens \
+    nodejs \
+    npm \
+    vim \
+    wget \
+    python3-pip \
+    net-tools \
+    iputils-ping \
+    terraform \
+    awscli \
+    default-jre \
+    default-jdk \
+    maven \
+    helm \
+    ufw \
+    golang-go \
+    && rm -rf /var/lib/apt/lists/*
+
+# Set the default directory
+WORKDIR /BUILDER
+
+# Create environment variables (APP_NAME and ENV) entered by the user at build time
+ARG APP_NAME
+ARG ENV
+ENV TEAM=Devops
+
```

Comment:
you are missing additional environments here. Please add them as per the requirement